core/domain/stats_domain_test.py

```diff
@@ -385,6 +510,54 @@ def test_validate(self):
             'Expected exp_id to be a string, received %s' % (type(5)))):
             exp_issues.validate()
 
+    def test_validate_exp_version(self):
+        exp_issues = stats_domain.ExplorationIssues(
+            'exp_id1', 1, [
+                stats_domain.ExplorationIssue.from_dict({
+                    'issue_type': 'EarlyQuit',
+                    'issue_customization_args': {
+                        'state_name': {
+                            'value': 'state_name1'
+                        },
+                        'time_spent_in_exp_in_msecs': {
+                            'value': 200
+                        }
+                    },
+                    'playthrough_ids': ['playthrough_id1'],
+                    'schema_version': 1,
+                    'is_valid': True})
+                ])
+        exp_issues.validate()
+
+        exp_issues.exp_version = 'invalid_version'
+        with self.assertRaisesRegexp(utils.ValidationError, (
+            'Expected exp_version to be an int')):
+            exp_issues.validate()
+
+    def test_validate_unresolved_issues(self):
+        exp_issues = stats_domain.ExplorationIssues(
+            'exp_id1', 1, [
+                stats_domain.ExplorationIssue.from_dict({
+                    'issue_type': 'EarlyQuit',
+                    'issue_customization_args': {
+                        'state_name': {
+                            'value': 'state_name1'
+                        },
+                        'time_spent_in_exp_in_msecs': {
+                            'value': 200
+                        }
+                    },
+                    'playthrough_ids': ['playthrough_id1'],
+                    'schema_version': 1,
+                    'is_valid': True})
+                ])
+        exp_issues.validate()
+
+        exp_issues.unresolved_issues = 0
+        with self.assertRaisesRegexp(utils.ValidationError, (
+            'Expected unresolved_issues to be a list')):
+            exp_issues.validate()
+
 
 class PlaythroughTests(test_utils.GenericTestBase):
```

Comment:
Please make sure all cases covered by [PlaythroughAudit](https://github.com/oppia/oppia/blob/7cdd42cb5878b235851f9ecb10e195a347f7a7a8/core/domain/stats_jobs_one_off.py#L42) are covered.

Reply:
I think this is supposed to be a part of stats_jobs_one_off_test PR. Is it OK to do it there as this PR only aims to achieve 100% coverage for stats_domain and stats_services

Reply:
I mean, make sure that the same errors `PlaythroughAudit` looks for are added as tests here (for the sake of code health, not 100% coverage, we need to make tests exhaustive).

Reply:
I think they can be a part of https://github.com/oppia/oppia/issues/6853?

Reply:
Resolved offline